mandatory/1-syntax-errors.js

```diff
@@ -24,19 +25,25 @@ To run these tests type `node 1-syntax-errors.js` into your terminal
 ===================================================
 */
 
-const util = require('util');
+const util = require("util");
 
 function test(test_name, actual, expected) {
-    let status;
```

Comment:
You edited the test runs???

Reply:
No I never thouched those Unit Tests written the